gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/indexed/IndexedCSVDataSource.java

```diff
@@ -82,9 +81,17 @@ public IndexedCSVDataSource(String csvPath, GradoopFlinkConfig conf, Configurati
     this.hdfsConfig = hdfsConf;
   }
 
+  /**
+   * Will use a single graph head of the collection as final graph head for the graph.
+   * Issue #1217 (https://github.com/dbs-leipzig/gradoop/issues/1217) will optimize further.
```

Comment:
I think this should be in a `@see` tag, but you can leave it as is, since this is copied from another comment